plugin/acl/README.md

```diff
@@ -0,0 +1,98 @@
+# acl
+
+*acl* - enforces basic access control policies and prevents unauthorized access to protected servers.
+
+## Description
+
+With `acl` enabled, users are able to filter DNS queries with filtering rules, i.e. allowing authorized queries to recurse or blocking unauthorized queries towards protected DNS zones.
```

Comment:
above you say `servers` here `zones`. Is it both?

Reply:
fixed.